apps/expo-go/ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.h

```diff
@@ -27,24 +25,25 @@ typedef enum EXReactAppManagerStatus {
 - (void)reactAppManagerDidInvalidate:(EXReactAppManager *)appManager;
 
 @end
-
-@interface EXReactAppManager : NSObject <RCTBridgeDelegate, EXAppFetcherDataSource>
+ 
+@interface EXReactAppManager : NSObject <EXAppFetcherDataSource>
 
 - (instancetype)initWithAppRecord:(EXKernelAppRecord *)record initialProps:(NSDictionary *)initialProps;
-- (void)rebuildBridge;
+- (void)rebuildHost;
 - (void)invalidate;
 
 // these are piped in from the view controller when the app manager is waiting for a bundle.
 - (void)appLoaderFinished;
 - (void)appLoaderFailedWithError:(NSError *)error;
 
 @property (nonatomic, assign) BOOL isHeadless;
-@property (nonatomic, readonly) BOOL isBridgeRunning;
+@property (nonatomic, readonly) BOOL isHostRunning;
 @property (nonatomic, readonly) EXReactAppManagerStatus status;
 @property (nonatomic, readonly) UIView *rootView;
 @property (nonatomic, readonly) NSString *scopedDocumentDirectory;
 @property (nonatomic, readonly) NSString *scopedCachesDirectory;
-@property (nonatomic, strong) id reactBridge;
+@property (nonatomic, strong) id reactHost;
+@property (nonatomic, strong) RCTAppDelegate *reactAppInstance;
```

Comment:
We need to hold on to the delegate or it will deallocate. I went with `reactAppInstance` for the name because that's what we are trying to control but we can't access the `RCTInstance` or `RCTHost` without going through the `RCTAppDelegate` and it's `RCTRootViewFactory`